mackerel-plugin-aws-kinesis-streams/lib/aws-kinesis-streams.go

```diff
@@ -0,0 +1,245 @@
+package mpawskinesisstreams
+
+import (
+	"errors"
+	"flag"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/cloudwatch"
+	mp "github.com/mackerelio/go-mackerel-plugin-helper"
+)
+
+const (
+	namespace          = "AWS/Kinesis"
+	metricsTypeAverage = "Average"
+	metricsTypeSum     = "Sum"
```

Comment:
[may]This metricType seems to be unused in this plugin. You can remove it.

If you keep `metricsTypeSum`, it would be good to add `Samples` statistic. According to official documents, some metrics have `Samples` statistic.

- http://docs.aws.amazon.com/ja_jp/streams/latest/dev/monitoring-with-cloudwatch.html